CFStats/CFUserInterface/UiModels/ApiHandlerModels.cs

```diff
@@ -11,4 +11,13 @@ public class ProblemModel
         public int wrongAttempts { get; set; }
         public int correctAttempts { get; set; }
     }
+
+    public class ContestModel
+    {
+        public string ContestId { get; set; }
+        public string ContestName { get; set; }
```

Comment:
Make ratingchange and rank as int variable

Reply:
done